saleor/graphql/product/types.py

```diff
@@ -286,6 +286,7 @@ class Product(CountableDjangoObjectType):
         Money,
         description=dedent("""The product's base price (without any discounts
         applied)."""))
+    tax_rate = TaxRateType(description='A type of tax rate.')
```

Comment:
The `a type of...` is sounding weird to me.
What do you think of: `The tax rate type.` ?

Reply:
Welp @maarcingebala just merged it :laughing: